src/analyzer/interactor/RequestModel.java

```diff
@@ -0,0 +1,37 @@
+package analyzer.interactor;
+
+import com.sun.istack.internal.NotNull;
+
+/**
+ * Is used in Controller class to get finances from the defined provider
+ * in order to use it later for analyzing.
+ *
+ * @author Lizaveta Havina
+ */
+public final class RequestModel {
```

Comment:
You probably got RequestModel concept wrong )
Request model is just a class which represents some configuration parameters for analysis.
Analyzer's analyze() method accepts RequestModel as a parameter and returns a Report.
Object of RequestModel is created by Controller's client

Reply:
That's exactly the thing I wanted to ask you, cause I doubted. Does that mean that RequestModel holds only field from_where_we_d_like_to_get_data and is created together with Presenter and Analyzer by Controller's client?